admin_manual/configuration/server/occ_command.rst

```diff
@@ -1779,8 +1816,7 @@ You may also use ``password-from-env`` to reset passwords:
 ::
 
  export OC_PASS=newpassword
- su -s /bin/sh www-data -c 'php occ user:resetpassword --password-from-env 
-   layla'
+ su -s /bin/sh www-data -c 'php occ user:resetpassword --password-from-env layla'
```

Comment:
https://github.com/owncloud/documentation/pull/4119/files#diff-0380163502e27936a0c96cf3eb64fe58R1543 uses `sudo`. How about using it here as well?